src/towncrier/_writer.py

```diff
@@ -6,12 +6,19 @@
 affecting existing content.
 """
 
+from __future__ import annotations
+
 from pathlib import Path
 
 
 def append_to_newsfile(
-    directory, filename, start_string, top_line, content, single_file
-):
+    directory: str,
+    filename: str,
+    start_string: str,
+    top_line: str,
+    content: str,
+    single_file: bool = True,
```

Comment:
Please don't re-add defaults to private APIs. 😅

```suggestion
    single_file: bool,
```

Reply:
Ah, that one slipped through when I rebased my changes on `trunk`.